index.html

```diff
@@ -111,8 +112,65 @@ <h3 class="card-heading">Multi-Post Stories Gain + Glory</h3>
   </section>
   <!-- About section  -->
   <section id="about">
+    <img src="assets/images/ab1.png" alt="" id="about-img-1">
```

Comment:
- [ ] [OPTIONAL] Good job making the changes requested, however I think it would be better if you use css background image for images which are decoration and not part of the content like this one, _'The section has the proper background image (decorations are always backgrounded images, not `img` tags)'_